git-hooks/commit-msg.py

```diff
@@ -54,6 +54,9 @@ def __init__(self, msg):
         self.subject = self.lines[0].rstrip() if self.lines else ""
         self.body = self.lines[1:]
 
+        # Strip special keywords used by `git rebase --autosquash`
+        self.subject = re.sub(r"^(fixup|squash)! ", "", self.subject)
```

Comment:
Hi @jimporter, thanks for the contribution! The change looks good to me -- I just have a small request. Would you mind adding a test case for it?

Reply:
Thanks for the catch. I didn't notice the tests farther down in the file. I've now added a test case.